rules/use-iapimarker-with-webapplicationfactory/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you use IApiMarker with WebApplicationFactory?
-uri: do-you-use-iapimarker-with-webapplicationfactory
```

Comment:
Hey @DevStarOps 

You forgot adding the old link to "redirects"

Let me know if in doubt ;)

Reply:
Added, thanks 😊